Makefile

```diff
@@ -0,0 +1,38 @@
+COMMIT_HASH = $(shell git log -n 1 --pretty=format:"%H")
+
+clean:
+		rm -rf ./env ./dist ./build ./artifacts ./dbsnap_verify.egg-info
+
+### Python2 ###
+
+test: clean virtualenv
+		. env/bin/activate && python setup.py pytest
+
+virtualenv:
+		virtualenv ./env
+
+build: clean virtualenv
+		. env/bin/activate && python setup.py install
+
+build-lambda: build
+	    mkdir ./artifacts
+		cp aws_lambda.py ./dist
+		cp -rf env/lib/python*/site-packages/* ./dist
+		cd ./dist && zip -r "../artifacts/lambda-dbsnap-verify-$(COMMIT_HASH).zip" .
+
+### Python3 ###
+
+test3: clean venv
+		. env/bin/activate && python setup.py pytest
+
+venv:
+		python3 -m venv env && env/bin/pip install setuptools --upgrade
```

Comment:
@phobologic this is what I was talking about during yesterday's stand up.

Reply:
Yeah, the pipenv stuff handles a bit more than this - it's basically looking to be bundler, which is useful (see their use of Pipfile).  This just replaces virtualenv it looks like.